corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -48,7 +48,11 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
                 options = {
                     success: function (parsedMenus, response) {
                         if (response.submitResponseMessage) {
-                            FormplayerFrontend.trigger('showSuccess', gettext(response.submitResponseMessage));
+                            let message = response.submitResponseMessage;
```

Comment:
Does this change only affect clickable icons response or a normal form submission response as well ? 

I am failing to understand from the PR why the change is needed and why it only affects web users ?

Reply:
This change removes the form and export links if they are present because broken/partial links are being displayed on clickable icon form submissions.
If the clickable icons form is submitted by a web user the links are appended to the success message here https://github.com/dimagi/commcare-hq/blob/d37138cacbe50343c3255d938c3f91b708d934ad/corehq/form_processor/submission_post.py#L172-L173

>Does this change only affect clickable icons response or a normal form submission response as well ?

This only affects clickable icons responses. Other form submissions get the appropriate working links.

Reply:
>  because broken/partial links are being displayed on clickable icon form submissions.

Do we know why though ? Can you give an example of broken link ?

Reply:
It's not clear yet why this is happening. I'm planning to create a ticket to follow up on it.

the broken link messages look like this:
`'Copy of switch fav' successfully saved! You submitted [this form\](/a/rc-local/reports/form_data/97783c46-fa68-427f-987b-e6859d98366g/). Click to export your [form data\](/a/rc-local/data/export/custom/form/). '

Reply:
is the issue that these are not complete links but only relative ?

Reply:
yes, I was saying "broken", but incomplete is more accurate.